src/k8-client/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2018"
 name = "k8-client"
-version = "5.5.1"
+version = "5.5.2"
```

Comment:
since this is new feature, this should bump up minor version: `5.6.0`

Reply:
done